tests/Unit/PointwiseFunctions/AnalyticSolutions/CMakeLists.txt

```diff
@@ -2,8 +2,8 @@
 # See LICENSE.txt for details.
 
 add_subdirectory(Burgers)
-add_subdirectory(EinsteinSolutions)
+add_subdirectory(Einstein)
```

Comment:
This isn't done until the next commit. Could you move it into the rename commit for einstein solutions?